index.bs

```diff
@@ -583,36 +583,39 @@ interface ML {
 };
 </script>
 
-The {{ML/createContext()}} method steps are:
-1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, then throw a "{{SecurityError!!exception}}" {{DOMException}} and abort these steps.
-1. Let |promise| be [=a new promise=].
-1. Let |context| be a new {{MLContext}} object.
-1. Switch on the method's first argument:
-    <dl class=switch>
-    <dt>{{MLContextOptions}}
-    <dd>Set |context|.{{[[contextType]]}} to [=default-context|default=].
-    <dd>Set |context|.{{[[deviceType]]}} to the value of {{MLContextOptions}}'s {{deviceType}}.
-    <dd>Set |context|.{{[[powerPreference]]}} to the value of {{MLContextOptions}}'s {{powerPreference}}.
-
-    <dt>{{GPUDevice}}
-    <dd>Set |context|.{{[[contextType]]}} to [=webgpu-context|webgpu=].
-    <dd>Set |context|.{{[[deviceType]]}} to "[=device-type-gpu|gpu=]".
-    <dd>Set |context|.{{[[powerPreference]]}} to "[=power-preference-default|default=]".
-    </dl>
-1. Issue the following steps to a separate timeline:
-    1. If the User Agent can support the |context|.{{[[contextType]]}}, |context|.{{[[deviceType]]}} and |context|.{{[[powerPreference]]}}, then:
-        1. Set |context|.{{MLContext/[[implementation]]}} to an implementation supporting |context|.{{[[contextType]]}}, |context|.{{[[deviceType]]}} and |context|.{{[[powerPreference]]}}.
-        1. [=Resolve=] |promise| with |context|.
-    1. Else:
-        1. [=Resolve=] |promise| with a new {{NotSupportedError}}.
-1. Return |promise|.
-
 ### Permissions Policy Integration ### {#permissions-policy-integration}
 
 This specification defines a <a>policy-controlled feature</a> identified by the
 string "<code><dfn data-lt="webnn-feature">webnn</dfn></code>".
 Its <a>default allowlist</a> is <code>'self'</code>.
 
+### The {{ML/createContext()}} method ### {#api-ml-createcontext}
+The {{ML/createContext()}} method steps are:
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, return [=a new promise=] [=rejected=] with a "{{SecurityError}}" and abort these steps.
```

Comment:
"[SecurityError](https://webidl.spec.whatwg.org/#securityerror)" [DOMException](https://webidl.spec.whatwg.org/#dfn-DOMException)

Reply:
Another convention round.
SecurityError is a DOMException by [definition](https://webidl.spec.whatwg.org/#idl-DOMException-error-names).
So, using the linked form 
```
{{SecurityError}}
```
without the quotes and without DOMException it would unambiguously refer to a "SecurityError" DOMException :)  

However, even if not the shortest, we should stick to the [Web IDL convention](https://webidl.spec.whatwg.org/#idl-exceptions):

> [Simple exceptions](https://webidl.spec.whatwg.org/#dfn-simple-exception) can be created by providing their [error name](https://webidl.spec.whatwg.org/#dfn-exception-error-name). A [DOMException](https://webidl.spec.whatwg.org/#idl-DOMException) can be [created](https://webidl.spec.whatwg.org/#dfn-create-exception) by providing its [error name](https://webidl.spec.whatwg.org/#dfn-exception-error-name) followed by [DOMException](https://webidl.spec.whatwg.org/#idl-DOMException). Exceptions can also be thrown, by providing the same details required to [create](https://webidl.spec.whatwg.org/#dfn-create-exception) one.

These rules are not used uniformly throughout the spec. 
The question is whether to fix that in this PR, or should it be a separate one, plus document the reasoning in an issue that links this discussion?